src/clzones.cpp

```diff
@@ -18,53 +18,110 @@
+                   zone_type( translate_marker_context( "vehicle parts", "Loot: V.Parts" ),
+                              translate_marker( "Destination for vehicle parts." ) ) );
+    types.emplace( zone_type_id( "LOOT_OTHER" ),
+                   zone_type( translate_marker( "Loot: Other" ),
+                              translate_marker( "Destination for other miscellaneous items." ) ) );
+    types.emplace( zone_type_id( "LOOT_FUEL" ),
+                   zone_type( translate_marker( "Loot: Fuel" ),
+                              translate_marker( "Destination for gasoline, diesel, lamp oil and other substances used as a fuel." ) ) );
+    types.emplace( zone_type_id( "LOOT_SEEDS" ),
+                   zone_type( translate_marker( "Loot: Seeds" ),
+                              translate_marker( "Destination for seeds, stems and similar items." ) ) );
+    types.emplace( zone_type_id( "LOOT_CHEMICAL" ),
+                   zone_type( translate_marker( "Loot: Chemical" ),
+                              translate_marker( "Destination for chemicals." ) ) );
     types.emplace( zone_type_id( "LOOT_SPARE_PARTS" ),
-                   zone_type( translate_marker_context( "spare parts", "Loot: S.Parts" ) ) );
+                   zone_type( translate_marker_context( "spare parts", "Loot: S.Parts" ),
+                              translate_marker( "Destination for spare parts." ) ) );
     types.emplace( zone_type_id( "LOOT_ARTIFACTS" ),
-                   zone_type( translate_marker( "Loot: Artifacts" ) ) );
-    types.emplace( zone_type_id( "LOOT_ARMOR" ), zone_type( translate_marker( "Loot: Armor" ) ) );
-    types.emplace( zone_type_id( "LOOT_FARMOR" ), zone_type( translate_marker_context( "filthy armor",
-                   "Loot: F.Armor" ) ) );
-    types.emplace( zone_type_id( "LOOT_WOOD" ), zone_type( translate_marker( "Loot: Wood" ) ) );
-    types.emplace( zone_type_id( "LOOT_IGNORE" ), zone_type( translate_marker( "Loot: Ignore" ) ) );
-    types.emplace( zone_type_id( "FARM_PLOT" ), zone_type( translate_marker_context( "plot of land",
-                   "Farm: Plot" ) ) );
+                   zone_type( translate_marker( "Loot: Artifacts" ),
+                              translate_marker( "Destination for artifacts" ) ) );
+    types.emplace( zone_type_id( "LOOT_ARMOR" ),
+                   zone_type( translate_marker( "Loot: Armor" ),
+                              translate_marker( "Destination for armor. Does include filthy armor if such zone is not specified." ) ) );
+    types.emplace( zone_type_id( "LOOT_FARMOR" ),
+                   zone_type( translate_marker_context( "filthy armor", "Loot: F.Armor" ),
+                              translate_marker( "Destination for filthy armor." ) ) );
+    types.emplace( zone_type_id( "LOOT_WOOD" ),
+                   zone_type( translate_marker( "Loot: Wood" ),
+                              translate_marker( "Destination for firewood and items that can be used as such." ) ) );
```

Comment:
Does this zone include items such as logs, 2x4s, etc?

Reply:
Anything that has FIREWOOD flag. Logs, 2x4s, pine boughs and few other similar items have it.

Reply:
All right. I thought the zone name is suggesting some place to put wooden crafting/construction materials, but to think of it maybe that role is already taken by the spare parts zone.